specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -11966,6 +12010,14 @@
         "useManagedDisks": {
           "description": "A value indicating whether managed disks should be used during failover.",
           "type": "string"
+        },
+        "targetProximityPlacementGroupId": {
+          "description": "The target proximity placement group Id.",
+          "type": "string"
+        },
+        "targetAvailabilityZone": {
+          "description": "The target availability zone.",
+          "type": "string"
         }
```

Comment:
please add at least one example to the spec showing what input to these properties looks like

Reply:
Examples were not updated as these new parameters are optional.